packages/apps-config/src/endpoints/testing.ts

```diff
@@ -315,10 +315,10 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
       }
     },
     {
-      info: 'oak-testnet',
-      text: t('rpc.test.oak', 'OAK Testnet', { ns: 'apps-config' }),
+      info: 'neumann',
+      text: t('rpc.test.oak', 'Neumann Network', { ns: 'apps-config' }),
       providers: {
-        'OAK Network': 'wss://rpc.testnet.oak.tech'
+        'OAK Network': 'wss://neumann.api.onfinality.io/public-ws'
```

Comment:
Replace `OAK Network` with `OnFinality`. We'll do a different `OAK Network` provider at a later time.